<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1595527266207' stop='1595527302040'>
    <name>Connection</name>
    <title>Connection</title>
    <test-cases>
        <test-case start='1595527268011' status='failed' stop='1595527301042'>
            <name>In-place</name>
            <title>In-place</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1595527268012' status='passed' stop='1595527280484'>
                    <name>Can start in-place connection</name>
                    <title>Can start in-place connection</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1595527280484' status='undefined'>
                    <name>Can assert that site is connected</name>
                    <title>Can assert that site is connected</title>
                    <attachments>
                        <attachment title='Test failed: Connection :: In-place' source='d93cc293-028d-4584-b54a-f4ff1725e4ed-attachment.png' type='image/png' size='245725'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Error: expect(received).toBeTruthy()
                
                Received: false</message>
                <stack-trace>Error: expect(received).toBeTruthy()
                
                Received: false
                    at toBeTruthy (/home/travis/build/Automattic/jetpack/tests/e2e/specs/connection.test.js:35:46)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>